test-cases/performance/scale-up/scale-up-base.yaml

```diff
@@ -0,0 +1,27 @@
+test_duration: 480
+
+k8s_loader_run_type: 'static'
+
+n_db_nodes: 3
+
+n_monitor_nodes: 1
+
+#GCE
+use_preinstalled_scylla: false  # required when using non standard instance type
+gce_root_disk_type_db: 'pd-ssd'
+gce_n_local_ssd_disk_db: 0
```

Comment:
worth adding a comment that this is "0" because we use c3-xxx-lssd which come with predfined number of local ssd.

Reply:
done